src/storage/mvcc/txn.rs

```diff
@@ -60,10 +60,23 @@ impl<S: Snapshot> MvccTxn<S> {
         })
     }
 
+    pub fn start_ts(&mut self, start_ts: u64) {
```

Comment:
It is best not to change 'start_ts' of the transaction after it is created. You can do this like:
```
let reader = MvccReader::new(...);
for command in commands {
       let mut txn = MvccTxn::from_reader(reader, ...);
       for key in command.keys() {
             if txn.prewrite(key, command.start_ts....)? {
                   ...
             }
       }
}
```